src/main/java/com/neighborhoodAssurance/service/AgencyServiceStub.java

```diff
@@ -7,7 +7,7 @@
 public class AgencyServiceStub implements IAgencyService {
 
 	@Override
-	public AgencyDTO fetchByOri(String ori) {
+	public AgencyDTO fetchByORI(String ORI) {

```

Comment:
While naming conventions don't explicitly say what to do with an acronym, I'd go with what the code reviewer suggested here... at least for the method name.  fetchByORI points to this being an acronym, not an abbreviation for origin.